docker-compose.yml

```diff
@@ -45,6 +45,10 @@ services:
       WIKIBASE_PINGBACK:
       MW_WG_ENABLE_UPLOADS:
       <<: *wikibase_extra_variables
+    labels:
+      - traefik.http.routers.service-wikibase.rule=Host(`mardi.formulasearchengine.com`,`wikibase.mardi.schubotz.org`,`mardi.schubotz.org`,`portal.mardi4nfdi.de`)
```

Comment:
You can drop the schubotz.org and formulasearchengine domain aliases.

Reply:
done. what about the formulasearchengine network aliases for the quickstatements and wdqs-frontend containers?

Reply:
This should be removed as well. Thank you